envs/example/ci-full/heat_stack.yml

```diff
@@ -0,0 +1,212 @@
+heat_template_version: 2013-05-23
```

Comment:
This is not needed for 31x. In master we have re-worked all ci envs to have heat templates.